bowling/game.cpp

```diff
@@ -0,0 +1,47 @@
+#include "game.hpp"
+
+#include <iostream>
+
+bool Game::getIsFinished() const {
+    return isFinished_;
+};
+
+void Game::checkGameProgress(const std::vector<std::pair<int, int>>& score) {
+    if (score.size() < 9) {
+        isFinished_ = false;
```

Comment:
not needed. The last else is the same case as this one.

Reply:
done